src/openzaak/components/documenten/api/tests/test_audittrails.py

```diff
@@ -76,7 +78,10 @@ def test_create_enkelvoudiginformatieobject_audittrail(self):
             informatieobject_create_audittrail.nieuw, informatieobject_data
         )
 
-    @skip("ObjectInformatieObject is not implemented yet")
+    @tag("oio")
+    @skip(
+        "Creation and destroy OIO via its resource is not supported until support for external API's is added."
```

Comment:
hrm, shouldn't this also just result in an audit trail entry? As in - the POST gets accepted, but nothing changes in the database, except the audit trail entry that gets created?

Reply:
I turned off `notifications` because objects are not created here and the notifications of creation ZIO and BIO would be just duplicated.
I turned off `audittrails` under the same logic, but I perhaps was mistaken. So `audittrails` are turned on again for OIO resource and the test case is unskipped

Reply:
Brought it under upstreams attention: https://github.com/VNG-Realisatie/gemma-zaken/issues/1283